src/xmipp/libraries/parallel/mpi_volumeset_align.cpp

```diff
@@ -0,0 +1,106 @@
+/***************************************************************************
+ *
+ * Authors:  Mohamad Harastani mohamad.harastani@upmc.fr
+ *	         Slavica Jonic slavica.jonic@upmc.fr
+ *           Carlos Oscar Sanchez Sorzano coss.eps@ceu.es
```

Comment:
Carlos did not (to the best of my knowledge) work on this code. Please remove him from header.

Reply:
He is there because he wrote the original code that I used as a template [here](https://github.com/I2PC/xmipp/blob/devel/src/xmipp/libraries/parallel/mpi_nma_alignment.cpp). I can remove him if you see it necessary.